code/index.html

```diff
@@ -2,21 +2,165 @@
 <html>
   <head>
     <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
+    <title>Project 2: Chipotle magazine</title>
+    <style>
+      @import url('https://fonts.googleapis.com/css2?family=Roboto:wght@100;700&display=swap');
```

Comment:
If you use the @import for fonts then it should be at the top of your CSS file 😊